javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java

```diff
@@ -62,25 +58,25 @@
 
     private PackageDeclaration pakage;
 
-    private List<ImportDeclaration> imports;
+    private List<ImportDeclaration> importsList;
 
-    private List<TypeDeclaration<?>> types;
+    private List<TypeDeclaration<?>> typesList;
 
     public CompilationUnit() {
     }
 
-    public CompilationUnit(PackageDeclaration pakage, List<ImportDeclaration> imports, List<TypeDeclaration<?>> types) {
+    public CompilationUnit(PackageDeclaration pakage, List<ImportDeclaration> importsList, List<TypeDeclaration<?>> typesList) {
         setPackage(pakage);
-        setImports(imports);
-        setTypes(types);
+        setImportsList(importsList);
+        setTypesList(typesList);
     }
 
-    public CompilationUnit(Range range, PackageDeclaration pakage, List<ImportDeclaration> imports,
-                           List<TypeDeclaration<?>> types) {
+    public CompilationUnit(Range range, PackageDeclaration pakage, List<ImportDeclaration> importsList,
+                           List<TypeDeclaration<?>> typesList) {
         super(range);
         setPackage(pakage);
-        setImports(imports);
-        setTypes(types);
+        setImportsList(importsList);
```

Comment:
We could have had a discussion whether we should use "imports" or "importList" - but not "importsList" please.